internal/jimm/watcher.go

```diff
@@ -343,9 +343,9 @@ func (w *Watcher) watchController(ctx context.Context, ctl *dbmodel.Controller)
 						machines++
 						cores += n
 					}
-					m.Cores = cores
-					m.Machines = machines
-					m.Units = int64(len(v.units))
+					// m.Cores = cores
```

Comment:
remove?

Reply:
i've done this on purpose, because i didn't remove the watcher yet, but I don't want them to update these fields to avoid tests being green because the watcher is still doing its job.